Add beautified maps

marisakaup opened the pull request:
Issue #20 

- [x] rearranged components
- [x] adapted label fonts and symbol sizes of the components
- [x] created one version in grey shades and one version highlighting CIS and SOCS3 components with colors

lknegendorf committed:
Delete BachmannPD_SBGNML03.sbgn

Different Tools do not support SBGN-ML 0.3 so far. No validation possible!

lknegendorf committed:
Delete BachmannPD.txt

No valid SIF specification possible for PD model.

lknegendorf committed:
Upload validated version

Newt file for valid model, SBGN-ML working in Vanted, Krayon, SBGNViz

lknegendorf committed:
Upload .xml file created by Vanted

Created after import of SBGN-ML file from Newt to Vanted

lknegendorf committed:
Upload validated SBGN-ML 0.2 Code

Validated using Newt, Vanted, Krayon and SBGNViz

lknegendorf committed:
Rename BachmannPD.sbgn to BachmannPD_newt.sbgn

lknegendorf committed:
Upload image files and SBGN-ML exported by different tools..

.. also .graphml file created by Vanted

marisakaup committed:
Added 2 versions of beautified newt-file

#20

marisakaup committed:
Merge branch 'gruppe3_sbgn_leo' into gruppe3_sbgn_marisa

marisakaup committed:
Revert "Merge branch 'gruppe3_sbgn_leo' into gruppe3_sbgn_marisa"

This reverts commit a3683f849c403a2c1d8885a8cf0f9a5f4311630b, reversing
changes made to cc615854fbfc070206234a6d435e4c9f860aeb1e.

marisakaup committed:
Updated SBGN map: Moved receptor components closer to membran

#20

marisakaup commented:
Suggest to delete yesterday´s colored map (model/sbgn/BachmannPD_tidyold_color.nwt). Not needed for archive.

lknegendorf commented:
Hey @marisakaup thank you for your work! In my opinion, the map looks great now, but I have one suggestion: Would it be possible to color also the arcs showing the influence of SOCS3 and CIS in the corresponding colors? That would improve the understanding even more I think.

For the files in the pull request I would suggest that you just include the colored files and delete all other files (then we would have the "untidy" files with validated SBGN-ML and the "tidy" colored files.
Just to clarify: Deleting does not mean that the files are gone because the are stored in the Git version control system anyways, so we could reconstruct them if we need them even if you delete the branch.

Best, Leo

lknegendorf commented on model/sbgn/BachmannPD_tidynew_color.nwt:
```diff
@@ -0,0 +1,1090 @@
+      <start x="-203.07948047782304" y="-1937.6177608361597"/>
+      <end x="-835.4082023335773" y="-1884.0349365442985"/>
+    </arc>
+    <arc id="nwtE_3d4edd18-e61f-4506-ac4c-3f97b0b2a5b6" class="inhibition" source="nwtN_5096a7f8-9215-47f2-8250-00dd6a3813e5" target="nwtN_042e2497-c05a-4c4d-9ef0-a60ef95ea14c">
```
Maybe also the arcs from SOCS3 and CIS could be colored

marisakaup replied:
Please review updated map c4da95a

lknegendorf commented on model/sbgn/BachmannPD_tidynew_grey.nwt:
```diff
@@ -0,0 +1,1074 @@
+<?xml version='1.0' encoding='UTF-8' standalone='yes'?>
```
Suggest to delete this file

lknegendorf replied:
See comment above

marisakaup replied:
Done; commit fa75906

marisakaup committed:
Deleted maps in grey shades

#20

marisakaup committed:
Updated complete map and added simplified version

#20

- Added color for arcs related to SOCS3 and CIS
- Added simplified version with summarized receptors

marisakaup committed:
Implemented request changes from review

#20

- removed color for arcs nvolved in the production of SOCS3 or CIS
- created new PNG and SVG files
- deleted simplified version (receptors summarized)
- deleted old colored version that showed different colors for compartements

marisakaup committed:
Changed file names in _beautified
